innopoints/views/project.py

```diff
@@ -141,11 +142,15 @@ def create_project():
     new_project.creator = current_user
     new_project.moderators.append(current_user)
 
-    try:
-        for new_activity in new_project.activities:
-            new_activity.project = new_project
+    for new_activity in new_project.activities:
+        new_activity.project = new_project
 
-        db.session.add(new_project)
+    moderation = Activity(name='Moderation', internal=True, working_hours=0)
```

Comment:
Since the moderation activity is created with the project while it is in the draft stage, the `../publish` endpoint should check for at least 1 non-internal activity, since it now just checks for non-empty list